src/reducers/demo.ts

```diff
@@ -1,19 +1,65 @@
 import { createSlice } from "@reduxjs/toolkit";
 
+// TODO: from the looks it needs this states, update once demo flow is more confirmed
```

Comment:
what do you mean by this states?

Reply:
the redux state, im not sure whats necessary, so i put what i think is required, can update this part once we know what to put in to store.

Reply:
👌